mkdocs/overrides/devbook/transactions/messages.mjs

```diff
@@ -0,0 +1,164 @@
+	signerPublicKey: senderKeyPair.publicKey.toString(),
+	deadline: timestamp.addHours(2).timestamp,
+	recipientAddress: recipientAddress.toString(),
+	mosaics: [{
+		mosaicId: generateMosaicAliasId('symbol.xym'),
+		amount: 1_000_000n	// 1 XYM
+	}],
+	message: plainMessage
+});
+transaction.fee = new models.Amount(feeMult * transaction.size);
+
+// Sign and announce the transaction
+let signature = facade.signTransaction(senderKeyPair, transaction);
+let jsonPayload = facade.transactionFactory.static.attachSignature(
+	transaction, signature);
+let transactionHash = facade.hashTransaction(transaction).toString();
+console.log('Transaction hash:', transactionHash);
+
+let announceResponse = await fetch(`${NODE_URL}/transactions`, {
+	method: 'PUT',
+	headers: { 'Content-Type': 'application/json' },
+	body: jsonPayload
+});
+console.log('Plain message transaction announced\n');
+
+// Wait a moment before sending the next transaction
+await new Promise(resolve => setTimeout(resolve, 2000));
+
+// ===== ENCRYPTED MESSAGE =====
+console.log('=== Sending Encrypted Message ===');
+
+// Fetch updated network time
+const timeResponse2 = await fetch(`${NODE_URL}${timePath}`);
+const timeJSON2 = await timeResponse2.json();
+timestamp = new NetworkTimestamp(
+	timeJSON2.communicationTimestamps.receiveTimestamp);
+
+// Create a message encoder with sender's key pair
+const messageEncoder = new MessageEncoder(senderKeyPair);
+
+// Encrypt the message using recipient's public key
+const secretMessage = new TextEncoder().encode(
+	'This is a secret message!');
+const encryptedPayload = messageEncoder.encode(
+	recipientPublicKey, secretMessage
+);
+console.log('Original message:',
+	new TextDecoder().decode(secretMessage));
+const hex = Buffer.from(encryptedPayload).toString('hex');
+console.log('Encrypted payload:', hex);
+
+// Build transfer transaction with encrypted message
+transaction = facade.transactionFactory.create({
+	type: 'transfer_transaction_v1',
+	signerPublicKey: senderKeyPair.publicKey.toString(),
+	deadline: timestamp.addHours(2).timestamp,
+	recipientAddress: recipientAddress.toString(),
+	mosaics: [{
+		mosaicId: generateMosaicAliasId('symbol.xym'),
+		amount: 1_000_000n	// 1 XYM
+	}],
+	message: encryptedPayload
+});
+transaction.fee = new models.Amount(feeMult * transaction.size);
+
+// Sign and announce the transaction
+signature = facade.signTransaction(senderKeyPair, transaction);
+jsonPayload = facade.transactionFactory.static.attachSignature(
+	transaction, signature);
+transactionHash = facade.hashTransaction(transaction).toString();
+console.log('Transaction hash:', transactionHash);
+
+announceResponse = await fetch(`${NODE_URL}/transactions`, {
+	method: 'PUT',
+	headers: { 'Content-Type': 'application/json' },
+	body: jsonPayload
+});
+console.log('Encrypted message transaction announced\n');
+
+// ===== DECRYPTING MESSAGE =====
+console.log('=== Decrypting Message ===');
+
+// Sender can decrypt using recipient's public key
```

Comment:
Yeah, but this is weird use case. We want to show the code that's currently commented-out in line 158.